src/main/java/catan/domain/model/game/GameBean.java

```diff
@@ -298,6 +320,14 @@ public void setNodes(Set<NodeBean> nodes) {
         this.nodes = nodes;
     }
 
+    public Integer getDiceValue() {
+        if (this.diceFirstValue != null && this.diceSecondValue != null) {
```

Comment:
we can avoid using "else" in this method. And make check in another way, to check if values are null, then return null, as it is exceptional case.